README.md

```diff
@@ -44,21 +44,24 @@ defmodule MyApp.Factory do
   def factory(:user, _attrs) do
     %User{
       name: "Jane Smith",
-      email: sequence(:email, &"email-#{&1}@example.com"),
```

Comment:
I typically prefer leaving these on because it makes it easier to reorder lines without having to add and remove commas. It also makes diffs less noisy.